.github/workflows/CI.yml

```diff
@@ -58,6 +60,12 @@ jobs:
         run: |
           conda install --yes -c omnia-dev openmm
 
+      - name: Install OpenMM nightly
+        shell: bash -l {0}
+        if: matrix.cfg.openmm == 'beta'
+        run: |
+          conda install -c conda-forge/label/openmm_rc -c conda-forge openmm
+
```

Comment:
Are there nightlies pushed to `conda-forge`? Or only betas/RCs?

Reply:
@mattwthompson So the "nightly" builds we do for openmm live at omnia-dev, but it doesn't use the `conda-build` infrastructure so it is kind of problematic. What I did was create a new label `conda-forge/label/openmm_dev` where I create new builds on demand.

Reply:
But I did make a mistake on the name of this step!